rules/imt-boardkeeper-debs.mk

```diff
@@ -0,0 +1,22 @@
+IMT_BK_MODULES = boardkeeper-modules_3.01_amd64.deb
+$(IMT_BK_MODULES)_URL = "http://192.168.157.83/yuriih/barefoot/bf_imt_debs/$(IMT_BK_MODULES)"
```

Comment:
We should add a common http server

Reply:
Sure. It was under discussion. We have a solution and it will be changed in next PRs

Reply:
for now it's like this but we will move to GitHub storage